chunk_test.go

```diff
@@ -636,3 +636,51 @@ func TestQueryArrowStreamDescribeOnly(t *testing.T) {
 		}
 	})
 }
+
+func TestRetainChunkWOHighPrecision(t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		query := "select 0"
+
+		ctx := WithArrowBatches(context.Background())
+
+		var rows driver.Rows
+		var err error
+
+		// must use conn.Raw so we can get back driver rows (an interface)
```

Comment:
In this repo we know this, we don't need to put this comment :)

Reply:
true, deleted!